### PR TITLE
Create and delete users and organizations in AactiveAdmin

### PR DESCRIPTION
**Pull request by ghost**
-Generate active admin views to create,  delete users and organizations models

### CODE REVIEW

**Review by accg14 — APPROVED** [review]

File: app/admin/users.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+ActiveAdmin.register User do
+  belongs_to :organization, optional: true
+  permit_params :ci, :name, :surname, :email, :organization_id, organization_attributes: %(id name)
```
Comment (fbalarini) [inline app/admin/users.rb]: i think ` organization_attributes ` is not necessary

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@
     let!(:user) { FactoryBot.build(:user, organization: organization) }
 
     it 'should let create new ones, with appropiate data' do
-      expect(user).to be_valid
+      expect(user). to be_valid
```
Comment (fbalarini) [inline spec/models/user_spec.rb]: Unnecessary whitespace

File: spec/factories/organizations.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 FactoryBot.define do
   factory :organization do
-    name { Faker::Name.name }
+    name 'MyString'
```
Comment (fbalarini) [inline spec/factories/organizations.rb]: Use Faker

**Review by fbalarini — APPROVED** [review]
